### ISSUE TITLE
more convenient menu access to spell-check corrections

### ISSUE THREAD

**Issue by brainchild0** [marktext/marktext#2082]
Currently, if a word is marked as spelled incorrectly, then the steps to substitute a corrections are 1) right-click on word, 2)  enter *Spelling* menu, 3) move cursor down to end of menu, and 4) select replacement word.

In fact, this common operation is available in many applications through as few as two steps, for example, 1) right-click on word, 2) select replacement word (from top of first-level popup menu).

I suggest simply moving the word suggestions to the top of the top-level menu.

**Comment by Fealthas** [comment]
Seconded. This app has such great UX, with the one exception of the spell check functionality.

**Comment by michael-mhrdd** [comment]
Agreed. For people using the feature frequently, the current implementation is quite impractical.

I suggest the following alternative: open a dedicated context menu only for the suggestions on **left**-click, and leave the current menu as is.

You could also provide a setting to turn off the additional menu, for users who prefer the status quo.
